.circleci/config.yml

```diff
@@ -385,27 +415,15 @@ jobs:
           name: "Run Cypress Integration Tests"
           command: "cd tdrs-frontend; yarn cy:run:ci"
       - run:
-          name:  "Test: Execute Frontend  OWASP ZAP Vulnerability Scan"
+          name: "Test: Build and Spin-up React app behind Nginx"
```

Comment:
Separated this from the `zap-scanner.sh` script into its own step so we could save some overhead of building the docker containers for nightly runs that don't need all containers up.